src/rdkafka_ssl.c

```diff
@@ -417,6 +417,123 @@ rd_kafka_transport_ssl_cert_verify_cb (int preverify_ok,
         return 1; /* verification successful */
 }
 
+#if OPENSSL_VERSION_NUMBER >= 0x1000200fL && !defined(LIBRESSL_VERSION_NUMBER)
+static int rd_kafka_transport_ssl_cert_fetch_cb(SSL *ssl, void *arg) {
+        rd_kafka_transport_t *rktrans = rd_kafka_curr_transport;
+        rd_kafka_broker_t *rkb;
+        rd_kafka_t *rk;
+        int r;
+        int i;
+        int ret;
+        int num_chain_certs;
+
+        rd_assert(rktrans != NULL);
```

Comment:
All variable defs need to go at the top of the scope before any code, for compatibility with <c99 compilers.

Reply:
Hah, didn't realise that's a thing that we still needed! Will fix this up.